mne/io/cnt/tests/test_cnt.py

```diff
@@ -37,11 +41,20 @@ def test_data():
     assert raw.info["meas_date"] is None
 
 
+@testing.requires_testing_data
+def test_new_data():
+    """Test reading raw cnt files with different header."""
+    with pytest.warns(RuntimeWarning, match="Could not parse meas date"):
+        raw = read_raw_cnt(input_fname=fname, header="new")
+
+    assert raw.info["bads"] == ["F8"]  # test bads
+
+
 @testing.requires_testing_data
 def test_compare_events_and_annotations():
     """Test comparing annotations and events."""
     with pytest.warns(RuntimeWarning, match="Could not parse meas date"):
-        raw = read_raw_cnt(fname)
+        raw = read_raw_cnt(fname_bad_spans)
```

Comment:
Shouldn't need to change this, right?

Reply:
Yes, that doesn't need to be changed. I was going to revert it but I am having trouble with the line endings.